testing/jormungandr-integration-tests/src/jormungandr/grpc/client_tests.rs

```diff
@@ -68,12 +60,15 @@ pub async fn handshake_sanity() {
 // L1006 Tip request
 #[tokio::test]
 pub async fn tip_request() {
-    let fixture = Fixture::default();
-    let (server, config) = fixture.bootstrap_node();
-    let client = Config::attach_to_local_node(config.get_p2p_listen_port()).client();
+    let setup =
+        setup::client::bootstrap(ConfigurationBuilder::new().with_slot_duration(9).to_owned())
+            .await;
 
-    let tip_header = client.tip().await;
-    let block_hashes = server.logger.get_created_blocks_hashes();
+    while setup.client.tip().await.chain_length() < 1.into() {
```

Comment:
This could wait indefinitely if the mock does not do the right thing?

The whole loop looks like it could be put into an utility function, but then this pattern occurs in other unchanged code, so let's leave it for later refactoring.

Reply:
correct, I will refactor it in an utility function with a timeout.